puppeteer-browser-pool.js

```diff
@@ -0,0 +1,67 @@
+/**
+ * Copyright 2018 Google Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * @author rahulkumar66@ (Rahul Kumar)
+ */
+
+/**
+* Demonstrates how to reuse browser instances across multiple tasks
+* resulting in improved load times for subsequent requests.
+* This has huge load time improvements in case of SPAs
+*/
+
+const genericPool = require('generic-pool');
+const puppeteer = require('puppeteer');
+
+const VIEWPORT = { width: 1028, height: 800, deviceScaleFactor: 2 };
```

Comment:
Sorry, we need linting.

- Can you remove the surrounding whitespaces after `{` and the first key name and the space before `}`.
- Use 2-space indentation everywhere.